bp_me/src/include/c/bp_common_me_if.h

```diff
@@ -109,6 +109,7 @@ typedef enum {
 
```

Comment:
So one thing I was thinking about, that I saw somewhere else.  It should be totally possible to generate the C headers from the Verilog headers, just with a grep.  Or a python script if it turns out to be difficult
#define -> `define, etc.

Reply:
I've tried this before, but not everything is straightforward / 1:1 mapping. There are some oddities that make it more complicated than just sed or grep. So, for now I've been maintaining manually. It would be nice to auto-generate though; I'll add it to my future To Do list.

Reply:
Sounds good!